Tests/myAccountTests.js

```diff
@@ -0,0 +1,97 @@
+var myAccountPage = require("../Pages/myAccountPage"),
+    homePage = require("../Pages/homePage"),
+    loginPage = require('../Pages/loginPage'),
+    data = require("../Data/data.js"),
+    UITasks = require("../Pages/UITasks"),
+    itemPage = require("../Pages/itemPage");
+    
+browser.waitForAngularEnabled(false);
+browser.ignoreSynchronization = true;
+browser.manage().window().maximize();
+
+describe("016: My Account", function(){
+    beforeEach(() => {
+        homePage.openPageURL(data.homepageLink)
+            .then(() => homePage.clickOnLinks(data.loginLinkTitle))
+            .then(() => loginPage.logIn(data.userTester2,data.passwordTester1))
+            .then(() => myAccountPage.clickOnLinks(data.myAccountTitle));
+    })
+    it("001: User is able to open 'Profile' page", function(){
+        myAccountPage.clickOnLinks(data.profileLinkTitle)
+            .then(() => myAccountPage.validateElement(data.profileHeadingTitle))
+    }) 
+    it("002: User isn't able to check 'Profile' if he isn't logged in", function(){
+        UITasks.checkMyAccountSection(data.profileLinkTitle)
+    })
+    it("003: User is able to open 'Your bids' page", function(){
+        myAccountPage.clickOnLinks(data.yourBidsTitle)
+            .then(() => myAccountPage.validateElement(data.yourBidsTable))
+    })
+    it("004: User isn't able to check 'Your bids' if he isn't logged in", function(){
+        UITasks.checkMyAccountSection(data.yourBidsTitle)
+    })
+    it("005: User is able to open 'Settings' page", function(){
+        myAccountPage.clickOnLinks(data.settingsTitle)
+            .then(() => myAccountPage.validateElement(data.settingsTitle))
+    })
+    it("006: User isn't able to open 'Settings' page if he isn't logged in", function(){
+        UITasks.checkMyAccountSection(data.settingsTitle)
+    })
+    it("007: User is able to open 'Profile' page from 'Your Bids' section", function(){
+        myAccountPage.clickOnLinks(data.yourBidsTitle)
+            .then(() => myAccountPage.clickOnElement(data.profileButton))
+            .then(() => myAccountPage.validateElement(data.profileHeadingTitle))
+    })
+    it("008: User is able to open 'Profile' page from 'Settings' section", function(){
```

Comment:
Since this part "  myAccountPage.clickOnLinks(data.yourBidsTitle)
            .then(() => myAccountPage.clickOnElement(data.profileButton))
            .then(() => myAccountPage.validateElement(data.profileHeadingTitle))" is repeating 7 times can you make one UItask from this ?

Reply:
I created one method for these tasks.